src/__tests__/unit/util/helpers.test.ts

```diff
@@ -0,0 +1,119 @@
+import {mergeObjects} from '../../../util/helpers';
+
+describe('util/helpers: ', () => {
+  describe('mergeObjects(): ', () => {
+    it('does not override input', () => {
+      expect.assertions(1);
+
+      const input = {
+        a: 1,
+        b: false,
+        c: 'testInput',
+      };
+
+      const defaults = {
+        c: 'testDefault',
+      };
+
+      const result = mergeObjects(defaults, input);
+      const expectedResult = {
+        a: 1,
+        b: false,
+        c: 'testInput',
+      };
+
+      expect(result).toEqual(expectedResult);
```

Comment:
```ts
const expectedResult = {
        a: 1,
        b: false,
        c: 'testInput',
      };
```
This expected result can be removed and input can be directly used in assertion, f.ex. `expect(result).toEqual(input);`